spec/spec_helper.rb

```diff
@@ -8,6 +8,10 @@
   SimpleCov.formatter = SimpleCov::Formatter::Codecov
 end
 
+# Make sure tests go through the 'bad theme' code in config/application.rb
+ENV['SHINYCMS_THEME'] = 'TEST'
+Dir.mkdir 'app/views/themes/TEST' unless Dir.exist? 'app/views/themes/TEST'
```

Comment:
Should this directory be cleaned up after the tests run? If so, consider configuring a RSpec `around` hook (or `before`+`after` hooks)  to create and delete the directory.